README.md

```diff
@@ -19,3 +19,29 @@ of the production JupyterHubs.
 ## Images
```

Comment:
Note the rest of the README is already wildly inaccurate (except for 'Branches'). Can you remove those too?